### ISSUE TITLE
Purpose of gitinfo2 warning about 'gitinfo2 found'?

### ISSUE THREAD

**Issue by ljrk0** [Hightor/gitinfo2#17]
Hi, is there any reason why gitinfo2 warns about finding the .gin-file?:

https://github.com/Hightor/gitinfo2/blob/55fe7ce3caabc0bb560afd8aa32f1681f1b6c3c8/gitinfo2.sty#L110
Ex.:

```
Package gitinfo2 Warning: gitinfo2 found : ../../../.git/gitHeadInfo.gin
(gitinfo2)
```

**Comment by longborough** [comment]
Hi. Sorry I haven't replied sooner. This is a very good question, to which I don't know the answer. Obviously I shouldn't be issuing warnings for normal events.

I vaguely remember, at the time, there was a reason for this, but I'm going to have to go back and try to discover why, Please bear with me meanwhile.

**Comment by ljrk0** [comment]
Hi, no problem -- it's no show-stopping bug anyway ^^

Maybe it was supposed to be shown when grumpy-option was given?

**Comment by longborough** [comment]
Hi, again. It's already fixed (shaky memory) and is now (in the next release, real soon now!) an Info message.

I didn't attach it to `grumpy`, as if you want it you can always find it. The purpose of `grumpy` is to stop the whole show rather than defaulting the tags if there's a problem retrieving the gitHeadInfo.gin file.

**Comment by ljrk0** [comment]
Ah, cool thanks for fixing this, have a nice day!

**Comment by longborough** [comment]
Fixed by commit 498d368 for release 2.0.5
